draft-cutler-httpbis-cookies-having-independent-partitioned-state.md

```diff
@@ -159,7 +159,9 @@ The following could be added as a new subsection of section 6.1 (Limits):
 {:quote}
 > User agents SHOULD enforce a separate per-domain limit for partitioned cookies for a particular cookie-partition-key.
 > This limit should be lower than the per-domain limit for cookies without the Partitioned attribute to prevent cookies set on different top-level sites from reaching implementation memory limits.
-> For example, a user agent MAY limit domains to 10 partitioned cookies per partition.
+> Since memory is the main concern, instead of limiting the number of cookies a domain may use per partition, a user agent MAY limit how many bytes a domain's cookies occupy on the user agent's device.
+> For example, a user agent MAY limit domains' cookies to only occupy 10 kilobytes, where the size of each cookie is the sum of the number of octets in the cookie-name and cookie-value.
+> The user agent MAY reject any new cookie that would cause a domain to go over its per partition 10 kilobyte limit.
```

Comment:
In an example, which is inherently non-normative, I'd use "might" instead of "MAY". The MAY in the line before this, about limiting by bytes instead of number, might still need to be normative, depending on what the rest of the spec says about how UAs can limit the amount of cookies: if the rest of the spec says they can only limit by the number of cookies, this needs the normative MAY to let them do it differently in this case.

Reply:
We actually have decided to evict the least-recently accessed cookie in Chrome to keep it consistent with the eviction scheme for the number of cookies. We also have decided to add an additional 180 cookie per-domain-per-partition limit in addition to the 10 KB limit.

For now, I'll just remove this line.